regimes/pl/pl.go

```diff
@@ -39,6 +45,21 @@ func New() *tax.Regime {
 		Validator:        Validate,
 		Calculator:       Calculate,
 		Categories:       taxCategories, // tax_categories.go
+		Corrections: []*tax.CorrectionDefinition{
+			{
+				Schema: bill.ShortSchemaInvoice,
+				Types: []cbc.Key{
+					bill.InvoiceTypeCorrective,
+				},
+				ReasonRequired: true,
+				Stamps: []cbc.Key{
+					StampProviderKSeF,
+				},
+				Extensions: []cbc.Key{
+					ExtKeyKSEFCorrection,
```

Comment:
If this extension field is required, it might also need adding to invoice validation.

Reply:
I added validation for the extension field with the tests.